canonical-web-and-design/jaas-dashboard
# General improvements to the table data displayed

## Done
- Set OS to the application series
- Update notes to messages
- Change CharmHub to Charmhub
- Add `title` to the unit ports 

## QA
- Check the demo
- Step through the changes listed above and confirm they are correct.

## Details
Fixes https://github.com/canonical-web-and-design/jaas-dashboard/issues/956
